app/javascript/components/set-service-ownership-form/index.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Grid } from 'patternfly-react';
+import MiqFormRenderer from '../../forms/data-driven-form';
+import { http } from '../../http_api';
+import { cleanVirtualDom } from '../../miq-component/helpers';
+import createSchema from './schema';
+
+class SetServiceOwnershipForm extends Component {
+  state = {
+    initialValues: {},
+  }
+
+  componentDidMount() {
+    cleanVirtualDom();
+    const { ownershipIds } = this.props;
+    this.loadInitialData(ownershipIds);
+  }
+
+  loadInitialData = objectIds =>
+    http.post('/service/ownership_form_fields', { object_ids: objectIds })
```

Comment:
Sorry, all those `/service/..` are wrong :(.

We need to be using the current controller, otherwise this fails for all the screens changing ownership for non-services.

Looks like the code is expecting the controller to be one of:

```
          when "orchestration_stack"
          when "service"
          when "vm_or_template", "vm_infra", "vm_cloud", "vm"
          when "miq_template"
```

Reply:
You *can* use `/${ManageIQ.controller}/ownership_form_fields` if it helps.